ui/components/app/assets/asset-list/asset-list-control-bar/asset-list-control-bar.tsx

```diff
@@ -169,6 +211,18 @@ const AssetListControlBar = ({
     });
   };
 
+  const onEnableAutoDetect = () => {
+    history.push(SECURITY_ROUTE);
+  };
+
+  const handleNftRefresh = () => {
+    if (isMainnet) {
```

Comment:
Same for adding linea mainnet check 🙏

Reply:
https://github.com/MetaMask/metamask-extension/pull/32020/commits/adc7e1b473f2b9d1674e82cd074f8b76a678ad4e